src/repository/storage/postgres/orm/sequelize/noteVisits.ts

```diff
@@ -71,6 +70,11 @@ export default class NoteVisitsSequelizeStorage {
     }, {
       tableName: this.tableName,
       sequelize: this.database,
+      /**
+       * Auto-generated timestamp attributes would be removed from the model
+       * (e.g. auto-generated created_at field conflicts with note.created_at when noteVisitModel is included)
```

Comment:
the main reason is not that they are conflicts (which is solvable). We just don't need them.
```suggestion
       * Disable auto-generated created_at and updated_at since we use own visited_at
```